components/announcement_bar/announcement_bar.jsx

```diff
@@ -312,27 +393,50 @@ export default class AnnouncementBar extends React.PureComponent {
                                 target='_blank'
                             >
                                 <FormattedMessage
-                                    id='error_bar.site_url.docsLink'
+                                    id='announcement_bar.error.site_url.docsLink'
                                     defaultMessage='Site URL'
                                 />
                             </a>
                         ),
                         link: (
                             <Link to='/admin_console/general/configuration'>
                                 <FormattedMessage
-                                    id='error_bar.site_url.link'
+                                    id='announcement_bar.error.site_url.link'
                                     defaultMessage='the System Console'
                                 />
                             </Link>
                         ),
                     }}
                 />
             );
+        } else if (message === AnnouncementBarMessages.EMAIL_VERIFICATION_REQUIRED) {
+            message = (
+                <React.Fragment>
+                    <FormattedHTMLMessage
+                        id={AnnouncementBarMessages.EMAIL_VERIFICATION_REQUIRED}
+                        defaultMessage='Check your email at {email} to verify the address. Cannot find the email? '
```

Comment:
nit: `defaultMessage` has trailing space?

Reply:
You are right/ I updated the translations to be more consistent. `Sending` and `Send again` now hold a leading space. Is this ok or is there a better approach?